docs/content-services/services/legal-hold.service.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: Legal Hold service
+Added: v6.10.0
+Status: Active
+Last reviewed: 2024-06-19
+---
+
+# [Legal Hold service](../../../lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts "Defined in legal-hold.service.ts")
+
+Manages holds for nodes.
+
+## Class members
+
+### Methods
+
+-   **getHolds**(filePlanId: `string`, options?: `ContentPagingQuery`): [`Observable`](http://reactivex.io/documentation/observable.html)`<[`Hold`](../../../lib/js-api/src/api/gs-core-rest-api/docs/Hold.md)`[]>`<br/>
+    updates permissions setting from a node.
```

Comment:
@DaryaBalvanovich this description is incorrect probably. It says something about updating permissions but function is getHolds.

Reply:
fixed